### PR TITLE
20% for 11

### PR DESCRIPTION
**Pull request by LukasReschke**
@MorrisJobke Care to give it a review?

### CODE REVIEW

File: src/Response.php [hunk 1]
```diff
@@ -58,20 +58,22 @@ private function getStableResponse(array $versions,
...
-					$counter += $chance;
-					if($instanceChance <= $counter) {
+					$counter -= $chance;
+					if($instanceChance <= (100 - $counter)) {
 						$newVersion = $newVersions[$chance];
 						break;
 					}
 				}
 
+				//var_dump($newVersion);
```
Comment (LukasReschke) [inline src/Response.php]: Kill it 🙈

Comment (MorrisJobke) [inline src/Response.php]: 🙈

File: tests/integration/features/update.feature [hunk 2]
```diff
@@ -44,18 +44,119 @@ Feature: Testing the update scenario of releases
...
+    And The received PHP version is "5.6.0"
+    And the installation mtime is "20"
+    And The received version is "9.1.3.2"
+    When The request is sent
+    Then The response is non-empty
+    And Update to version "11.0.1.2" is available
+    And URL to download is "https://download.nextcloud.com/server/releases/nextcloud-11.0.1.zip"
+    And URL to documentation is "https://docs.nextcloud.com/server/11/admin_manual/maintenance/upgrade.html"
+
+  Scenario: Updating an up-to-date non-staged Nextcloud 10.0.3 with PHP 5.6 on the stable channel
```
Comment (MorrisJobke) [inline tests/integration/features/update.feature]: Shouldn't this result in a 11.0.1?

**Comment by MorrisJobke** [comment]
Deployed :)

**Comment by MorrisJobke** [comment]
let's see what happens

**Pull request merged**